content-src/components/PreferencesPane/PreferencesPane.js

```diff
@@ -48,56 +48,58 @@ const PreferencesPane = React.createClass({
         </div>
         <div className="prefs-pane">
           <div className={classNames("sidebar", {hidden: !this.state.showPane})} ref="sidebar">
-            <div className="prefs-modal-inner-wrapper">
```

Comment:
The only thing that really changed in this file is that I added a `<div className="sidebar-inner-wrapper">` to wrap all this content.